### PR TITLE
Create RELEASE.md

### PR DESCRIPTION
**Pull request by neeraj-laad**
Initial release.md

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+1) The PR will be reviewed and merged
+1) The maintainers will then create a release, which triggers the `deploy` phase of the Travis build
+1) The `deploy` phase pushes an updated image of the asset to Docker Hub
+
+# Release schedule
+We anticipate that this asset will be release rarely - only when issues warrant a correction.
+
+# Dependencies
+
+This asset uses the Ubuntu (Debian) installer package of the Appsody CLI, which is produced by the Appsody CLI build process. That package is referenced by the `Dockerfile` contained in this repo and it is currently set to consume a specific release of the Appsody CLI. This release is configuresd as a Travis environment variable.
```
Comment (chilanti) [inline RELEASE.md]: Buildah requires the RPM (Fedora) installer.

**Review by chilanti — CHANGES_REQUESTED** [review]
The only change I'd suggest is that this one requires the RPM installer.

File: RELEASE.md (hunk 1 above)
Comment (neeraj-laad) [reply RELEASE.md]: Updated

**Review by chilanti — APPROVED** [review]
Good to go.

**Pull request merged**